source/ui/AMBeamlineCameraWidget.cpp

```diff
@@ -61,6 +63,7 @@ AMBeamlineCameraWidget::AMBeamlineCameraWidget(QWidget *parent, bool useOpenGlVi
 	crosshairThicknessSlider_->setValue(1);
 	chl->addSpacing(20);
 	chl->addWidget(lockCrosshairCheckBox_ = new QCheckBox("Lock position"));
+	chl->addWidget(recenterCrosshairPushbutton_ = new QPushButton("Recenter"));
```

Comment:
I really don't like this style of instantiating variables like that.  Same goes for the things around it.